Packs/CVE_2024_47575/Playbooks/playbook-CVE-2024-47575_-_FortiManager_Authentication_Bypass.yml

```diff
@@ -0,0 +1,542 @@
+          "y": 190
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 23a45588-c954-4d0c-8f0b-61d032408181
+    type: title
+    task:
+      id: 23a45588-c954-4d0c-8f0b-61d032408181
+      version: -1
+      name: Create and Tag Indicators
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "3"
+      - "4"
+      - "9"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 350
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 41432b0b-182a-425e-847d-0f294360c5d5
+    type: regular
+    task:
+      id: 41432b0b-182a-425e-847d-0f294360c5d5
+      version: -1
+      name: IP Indicators
+      description: commands.local.cmd.new.indicator
+      script: Builtin|||createNewIndicator
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      tags:
+        simple: FortiManager, CVE-2024-47575, FortiJump
+      type:
+        simple: IP
+      value:
+        simple: ${ExtractedIndicators.IP}
+      verdict:
+        simple: Malicious
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 490
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: a087634c-c1d3-4954-882d-051e8d896969
+    type: regular
+    task:
+      id: a087634c-c1d3-4954-882d-051e8d896969
+      version: -1
+      name: CVE Indicators
+      description: commands.local.cmd.new.indicator
+      script: Builtin|||createNewIndicator
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      tags:
+        simple: FortiManager, CVE-2024-47575, FortiJump
+      type:
+        simple: CVE
+      value:
+        simple: ${ExtractedIndicators.CVE}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 870,
+          "y": 490
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 47dddb71-adfa-42b3-892c-dd8bf0bc4694
+    type: title
+    task:
+      id: 47dddb71-adfa-42b3-892c-dd8bf0bc4694
+      version: -1
+      name: Threat Hunting
```

Comment:
I think can remove this section (double sections). maybe combine them into one? threat hunting - XQL  / threat hunting - XQL Hunting Queries / or just XQL Hunting Queries?

Reply:
Correct, Rapid pack related section header